src/els_hover_provider.erl

```diff
@@ -57,6 +57,33 @@ documentation(_M, _POI) ->
 %% @doc get the docs
 -spec get_docs(atom(), atom(), byte()) -> binary().
 get_docs(M, F, A) ->
+  %% Use apply and hard coded record so it can be compiled on older OTP's
+  try apply(code, get_doc, [M]) of
+    {ok, {docs_v1, _, _, <<"application/erlang+html">>, MDoc, _, _} = DocChunk}
+      when MDoc =/= none ->
+        %% Should probably write our own renderer with markdown
+        %% for now use stdlibs builtin shell renderer
+        case apply(shell_docs, render, [M, F, A, DocChunk]) of
+          {error, _R0} ->
+            docs_from_src(M, F, A);
+          FuncDoc ->
+            #{ kind => content_kind()
+             , value => unicode:characters_to_binary(FuncDoc)
+             }
+        end;
+    _R1 ->
+      docs_from_src(M, F, A)
+  catch error:undef ->
+      lager:info("err: ~p~n", [undef]),
```

Comment:
We should probably use a `warning` level here and a nicer error message. Also, no need for newline.

```suggestion
      lager:warning("Error fetching documentation. Using an old OTP version?", []),
```

Reply:
That is a debug printout used during development, so IMHO it should be removed.

Reply:
Fine with that, too.